bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/HueBindingConstants.java

```diff
@@ -86,7 +87,7 @@
     public static final String PROTOCOL = "protocol";
     public static final String USER_NAME = "userName";
 
-    // Light config properties
+    // Light and sensor config properties
```

Comment:
Just `// Configuration properties` as you have one for group thing type too now.